src/main.rs

```diff
@@ -168,6 +171,25 @@ fn handle_shp(file_path: &str, simplification: f64, is_area: bool) -> Vec<GridGe
         .collect()
 }
 
+fn handle_wkt(input_str: String) -> Vec<GridGeom<f64>> {
+    let wkt = Wkt::<f64>::from_str(&input_str).expect("");
+    wkt.items
+        .into_iter()
+        .filter_map(|s| try_into_geometry(&s).ok())
+        .flat_map(|geo| {
```

Comment:
I think `GridGeom::vec_from_geom` should be able to handle the cases here, so we could this instead of the two calls to `flat_map` currently

```rust
.flat_map(|g| GridGeom::<f64>::vec_from_geom(g, simplification, is_area))
```

Reply:
yeah! I saw.
Thanks!